controllers/sriovnetworknodepolicy_controller.go

```diff
@@ -61,8 +62,20 @@ func (r *SriovNetworkNodePolicyReconciler) Reconcile(req ctrl.Request) (ctrl.Res
 
 	reqLogger.Info("Reconciling")
 
+	policy := &sriovnetworkv1.SriovNetworkNodePolicy{}
+	err := r.Get(context.TODO(), req.NamespacedName, policy)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			reqLogger.Info("SriovNetworkNodePolicy resource not found. Ignoring since object must be deleted")
+			return ctrl.Result{}, nil
+		}
+		// Error reading the object - requeue the request.
+		reqLogger.Error(err, "Failed to get SriovNetworkNodePolicy")
+		return ctrl.Result{}, err
```

Comment:
If the policy is deleted, shall we return nil instead of trying indefinitely?

Reply:
We return nil at line 70.